views/pages/ChartsPage.jsx

```diff
@@ -79,12 +77,26 @@ const ChartsPage = () => {
   }, [users])
 
   return (
-    <div>
-      <ChartList
-        handleShareChart={handleShareChart}
-        chartList={chartList}
-        removeChart={removeChart}
-        onDuplicateChart={onDuplicateChart}
+    <Box sx={{ p: '20px' }}>
+      <PageHeader
+        title="Charts"
+        cta={
```

Comment:
what does cta stand for..? i just keep seeing Chicago Transit Authority

Reply:
haha, CTA stands for Call To Action